bin/shopify_api.py

```diff
@@ -8,6 +8,6 @@
 project_root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 sys.path.insert(0, project_root)
 
-with open(os.path.join(project_root, 'scripts', 'shopify_api.py')) as f:
-    code = compile(f.read(), f.name, 'exec')
+with open(os.path.join(project_root, "scripts", "shopify_api.py")) as f:
```

Comment:
Which check causes the quotes to change?

Reply:
that is done by [black](https://github.com/psf/black) which calls itself "the uncompromising code formatter". If we don't like this I can add `--skip-string-normalization` as an argument